docs/core-concepts/pages/editor-blocks.md

```
@@ -170,7 +184,7 @@ Generate or transform content directly within your pages using AI. The AI Block
 
 Type `/ai` to insert an AI Block. The block appears with a prompt input area and a dropdown to select the action type.
 
-### Action types
+#### Action types
```

markdownlint-cli2 (0.22.1)

[warning] Heading levels should only increment by one level at a time
Expected: h3; Actual: h4

(MD001, heading-increment)